SonataNotificationBundle.php

```diff
@@ -26,6 +27,10 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new NotificationCompilerPass());
 
         $this->registerFormMapping();
+
+        if (class_exists('Doctrine\Bundle\MongoDBBundle\DependencyInjection\Compiler\DoctrineMongoDBMappingsPass')) {
+            $container->addCompilerPass(new NotificationMappingCompilerPass());
```

Comment:
if this is only for Mongo, shouldn't the classname of the compiler pass reflect that?